docs/meshcloud.tenant.md

```diff
@@ -27,3 +27,35 @@ meshStack automatically derives [metadata tags](./meshcloud.metadata-tags.md) fo
 the meshCustomer it belongs to.
 
 Any update to tenant metadata (e.g. a change in payment method) triggers a new meshTenant reconciliation cycle.
+
+## Delete a meshTenant
+
+> Only users or users of [group](meshcloud.customer.md#user-groups) with the role [Customer Admin](meshcloud.customer.md#assign-meshcustomer-roles) have access to the administrative functionality described in this section.
```

Comment:
```suggestion
> Only users with the role [Customer Admin](meshcloud.customer.md#assign-meshcustomer-roles) or [Customer Owner](meshcloud.customer.md#assign-meshcustomer-roles) have access to the administrative functionality described in this section.
```

Reply:
I know groups would also work but I'd just keep it simple by mentioning 'users'.